src/kudu/consensus/leader_election.h

```diff
@@ -177,7 +177,8 @@ class FlexibleVoteCounter : public VoteCounter {
     int64_t term,
     std::map<std::string, int32_t>* region_unpruned_counts) const;
 
-  // Fetch the region corresponding to server UUID.
+  // Fetch the region corresponding to server UUID. Returns an empty string
+  // if UUID is not found. This could happen during a configuration change.
```

Comment:
why will it happen during configuration change? if we see an entry (uuid) in the configuration, we will always have a 
RaftPeerPB for it.

Reply:
Instance removal is an example. I have clarified in the comment.